dask_jobqueue/sge.py

```diff
@@ -74,3 +74,7 @@ def __init__(self,
         self.job_header = header_template % config
 
         logger.debug("Job script: \n %s" % self.job_script())
+
+    def _jobid_from_submit_output(self, out):
+        """ Get the unique identifier for a job from the submission status. """
+        return out.decode().split('.')[0].strip()
```

Comment:
For SGE I think you actually just need `out.decode().strip()` because `qsub -terse` only outputs the job id.